index.js

```diff
@@ -100,8 +111,11 @@ function pdIncidentDetail(id) {
  */
 function getServices(offset) {
   return pdServices(offset).then((result) => {
-    services = services.concat(result.body.services);
-    if (result.body.more) {
+    // Feature Flag
+    const resBody = USING_NEW_PD_BRIDGE ? result.body : result.body.services;
```

Comment:
Could 'result.body' or 'result.body.services' be undefined/null? Do we need to check?
 @ConalCosgrove

Reply:
good catch, will change now